network-api/networkapi/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -207,7 +207,7 @@ def test_serve_page_many_products_logged_in(self):
             buyersguide_factories.ProductPageFactory(parent=self.bg)
         products = ProductPage.objects.descendant_of(self.bg)
         self.assertEqual(products.count(), additional_products_count + 1)
-        query_number = 966
+        query_number = 961
```

Comment:
Not entirely sure why this query count has changed, looking for a second opinion here before committing this

Reply:
I think there is some connection between the random data generation and the exact ordering of fields in the models: https://github.com/MozillaFoundation/foundation.mozilla.org/blob/main/docs/workflow.md#visual-regression-testing

Maybe under the hood some fields have been added that change the random values we get. That coupled with the random assignment of categories to the product pages might explain this slight change in queries. 
https://github.com/MozillaFoundation/foundation.mozilla.org/blob/3340a0718df01c549ef8848e97f659c78469a446/network-api/networkapi/wagtailpages/factory/buyersguide.py#L140-L154

It could also be that Wagtail is performing some query more efficient now (not that this page would benefit much 😬